Namespace(withAug=True, AugDirect=20, method_name='SymDiGCN', GPUdevice=1, IsDirectedData=False, dataset='WikipediaNetwork/squirrel', undirect_dataset='PubMed', MakeImbalance=True, CustomizeMask=False, seed=100, K=2, heads=8, log_root='../logs/', log_path='test', data_path='../dataset/data/tmp/', epochs=1500, num_filter=2, p_q=0.95, p_inter=0.1, dropout=0.0, debug=False, new_setting=False, layer=2, lr=0.005, l2=0.0005, to_undirected=False, alpha=0.1, randomseed=-1, imb_ratio=100, n_layer=2, feat_dim=64, warmup=5, epoch=900, tau=2, max=False, no_mask=False, gdc='ppr')
CUDA Device Index: 1
12-30-14:37:03
excel_file_path is  TrueAug_SymDiGCN_PubMed12-30-14:37:03_undir.xlsx
Dataset is  PubMed() 
Chosen from DirectedData:  False
Num of bidrected edges: 44324, total num of edges: 88648
This is directed graph:  False
data_x torch.Size([19717, 500])
Beginning for split:  0 30-14:59:54
Traceback (most recent call last):
  File "/home/qj2004/magnet/src/GNNChangeTrainTest.py", line 648, in <module>
    main(args)
  File "/home/qj2004/magnet/src/GNNChangeTrainTest.py", line 368, in main
    edge_index1, edge_weights1 = get_appr_directed_adj(args.alpha, new_edge_index.long(),
  File "/home/qj2004/magnet/src/utils/edge_data.py", line 386, in get_appr_directed_adj
    L = (torch.mm(torch.mm(pi_sqrt, p_ppr), pi_inv_sqrt) + torch.mm(torch.mm(pi_inv_sqrt, p_ppr.t()), pi_sqrt)) / 2.0
RuntimeError: CUDA out of memory. Tried to allocate 4.17 GiB (GPU 1; 23.69 GiB total capacity; 18.22 GiB already allocated; 3.14 GiB free; 19.62 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
